c10/util/BFloat16-math.h

```diff
@@ -10,95 +11,188 @@ C10_CLANG_DIAGNOSTIC_IGNORE("-Wimplicit-float-conversion")
 
 namespace std {
 
-/// Used by vec256<c10::BFloat16>::map
-inline c10::BFloat16 acos(c10::BFloat16 a) {
+template <typename T>
+struct is_reduced_floating_point
```

Comment:
This has been defined in `ATen/cpu/vec/vec_base.h`, you may just include `<ATen/cpu/vec/vec.h>` to use it.